docs/4.x/generated/Piwik/Columns/Dimension.md

```diff
@@ -535,7 +535,7 @@ Returns the name of the plugin that contains this Dimension.
 
 - It returns a `string` value.
 - It throws one of the following exceptions:
-    - [`Exception`](http://php.net/class.Exception) &mdash; if the Dimension is not located within a Plugin module.
+    - `Piwik\Columns\Exception` &mdash; if the Dimension is not located within a Plugin module.
```

Comment:
This one actually throws `\Exception` and not `Piwik\Exception` and neither `Piwik\Columns\Exception` and the link went missing as well. Same issue seems to be happening many times.